### PR TITLE
[bluetooth.am43] AM43 Binding initial contribution

### PR DESCRIPTION
**Pull request by cpmeister**
This is a binding for the bluetooth controlled [AM43 Blind Drive Motor](https://www.a-okmotors.com/am-43-panel-japones/). These can be bought pretty cheaply (from china) which is why I felt having a binding for it would be a good idea. 
You can find details from my [community post](https://community.openhab.org/t/new-binding-am43-blind-drive-motor-binding/90272).
I have done considerable testing with it and the binding has proved exceedingly reliable, more so than the bluez binding that I've been testing it on.
You can find a prebuilt jar [here](https://github.com/cpmeister/openhab-addons/blob/blind-engine-snapshots/org.openhab.binding.bluetooth.am43-2.5.2-SNAPSHOT.jar)

### CODE REVIEW

File: bundles/org.openhab.binding.bluetooth.am43/src/main/java/org/openhab/binding/bluetooth/am43/internal/AM43Handler.java [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+
+    private @Nullable MotorSettings motorSettings = null;
+
+    public AM43Handler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.bluetooth.am43/src/main/java/org/openhab/binding/bluetooth/am43/internal/AM43Handler.java]: This method is deprecated. Please set the thing status to UNKNOWN here instead.

Comment (cpmeister) [reply bundles/org.openhab.binding.bluetooth.am43/src/main/java/org/openhab/binding/bluetooth/am43/internal/AM43Handler.java]: The parent handler class is ConnectedBluetoothHandler, so the super.initialize() is required.

Comment (kaikreuzer) [reply bundles/org.openhab.binding.bluetooth.am43/src/main/java/org/openhab/binding/bluetooth/am43/internal/AM43Handler.java]: Outch, you've got me there :-) You're obviously right, I missed that.

Comment (cpmeister) [reply bundles/org.openhab.binding.bluetooth.am43/src/main/java/org/openhab/binding/bluetooth/am43/internal/AM43Handler.java]: I don't blame you, the bluetooth binding is a unique case 😄

**Pull request merged**